laravel/the-basics/databases.html.md.erb

```diff
@@ -577,16 +577,19 @@ fly volumes create storage_vol --region ams --size 20 #GB
 
 <%= partial "docs/laravel/partials/volume_storage_warning" %>
 
-3) To fix the little storage-content-erasure issue, please go ahead and make a copy of your storage folder in a "backup" folder. You can name this directory "storage_".
+</li>
+<li>
+To fix the little storage-content-erasure issue, please go ahead and make a copy of your storage folder in a "backup" folder. You can name this directory "storage_".
 
 ```cmd
 cp -r storage storage_
 ```
 <small>
  You'll later use this folder to copy over its contents to the volumized storage folder.
 </small>
-
-4) Next create a [Startup Script](/docs/laravel/the-basics/customizing-deployments/) that will initialize the volumized storage folder's contents.
+</li>
```

Comment:
Similar to my first comment, a <br> or line separator here( between lines 590 and 591 ) would be really helpful, else items 4 and 5 will look squeezed together like so:
-------------------------------
![image](https://user-images.githubusercontent.com/33826441/211873723-af81520a-a701-42e0-9b39-32a2dbb96c01.png)

Reply:
I removed the "small" and added a br